crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -1016,14 +1016,14 @@ pub mod wasm_test {
 
         // run two zome calls concurrently
         // as the first zome calls, init and wasm compilation will happen and
-        // should take less than 10 seconds in debug mode
+        // should take less than 30 seconds in debug mode
```

Comment:
I can't fathom what in my changes could possibly have affected the wasm compile time, but I *did* need to bump this for this test to pass.

Reply:
This hasn't been known to be flaky AFAIK. Weird!

Reply:
+1 I've not seen this fail. Was it failing on CI as well as locally without the time increased?

This is where it would be good if I was further along with wind tunnel so we could check this without the distraction of performance in shared CI environments etc :(

Reply:
I added a benchmark for:
- binding a new listening socket
- creating a new connection to that socket
- sending a request
- and receiving a response

```
full_connect            time:   [203.87 µs 205.13 µs 206.25 µs]
                        Performance has regressed.
Found 5 outliers among 100 measurements (5.00%)
  1 (1.00%) low severe
  1 (1.00%) low mild
  3 (3.00%) high mild
```

If it can do all that in ~200 microseconds, I'm fairly confident at the very least in saying the websocket code isn't the problem here...

Reply:
Huh... I just switched it back to 10s locally and it passed... I'll run it in CI that way, maybe there was an interim bad state in my branch that since has been cleaned up somehow.

Reply:
Yep, passes in CI: https://github.com/holochain/holochain/actions/runs/8103573164/job/22148454507#step:4:2232 -- okay, nevermind, everything's good : )